spec/rails_helper.rb

```diff
@@ -0,0 +1,61 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
Can we get rid of the comments in this file?

They clutter the place up.

Reply:
Sure